mlir/lib/Conversion/GPUToSPIRV/GPUToSPIRVPass.cpp

```diff
@@ -108,6 +143,29 @@ void GPUToSPIRVPass::runOnOperation() {
     if (failed(applyFullConversion(gpuModule, *target, std::move(patterns))))
       return signalPassFailure();
   }
+  // In case of OpenCL, gpu.func in original gpu.module needs to replaced with
```

Comment:
nit: add a newline before a large block of code like this

Reply:
Done.